core/src/components/toggle/toggle.tsx

```diff
@@ -106,6 +99,18 @@ export class Toggle implements ComponentInterface {
     }
   }
 
+  private toggleChecked() {
+    const { checked, value } = this;
+
+    const isNowChecked = !checked;
+    this.checked = isNowChecked;
+
+    this.ionChange.emit({
+      checked: isNowChecked,
+      value: value,
```

Comment:
~ Shorthand syntax could be used here:

```suggestion
      value,
```